examples/sample-pipeline/sample-pipeline.cue

```diff
@@ -172,6 +172,8 @@ frsca: task: "deploy-using-kubectl": {
 				"s;__IMAGE__;$(params.imageUrl):$(params.imageTag);g",
 				"-e",
 				"s;__DIGEST__;$(params.imageDigest);g",
+				"-e",
+				"s;registry.registry/;localhost:5000/;g",
```

Comment:
isn't this also pointing to `ttl.sh` wouldn't this cause an issue?

Reply:
It's one entry of a long `sed` command, so if the `registry.registry/` string isn't seen, it doesn't do anything.